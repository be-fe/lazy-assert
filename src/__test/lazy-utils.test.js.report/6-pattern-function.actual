{}
  01 string, typeof === string : "ok"
  02 123, typeof === string : {}
    fail : 123
  03 {a: 2}, val.a > 1 : "ok"
  04 {a: 1}, val.a > 1 : {}
    fail : {}
      a : 1
  05 return first + last, result === first + last : "ok"
  06 return first, result === first + last : {}
    fail : "_[[[function]]]_"