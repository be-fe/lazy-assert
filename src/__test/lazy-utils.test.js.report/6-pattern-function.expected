{}
  01 string, typeof === string : "ok"
  02 123, typeof === string : {}
    fail : 123
  03 {a: 2}, val.a > 1 : "ok"
  04 {a: 1}, val.a > 1 : {}
    fail : {}
      a : 1
  05 return first + last, result === first + last : "ok"
  06 return first, result === first + last : {}
    fail : "_[[[function]]]_"
  07 [{a: 1}, {a: 2}, "a", 123] : []
    0 : {}
      fail : {}
        a : 1
    1 : "ok"
    2 : {}
      fail : "a"
    3 : {}
      fail : 123